app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/optionsStepForm/itemsList/itemsList.jsx

```diff
@@ -33,8 +33,8 @@ export const ItemsList = ({
 }) => {
   const [showErrorLogs, setShowErrorLogs] = useState(false);
```

Comment:
Why not just set the default `showErrorLogs` to `true`?
In any case, they will be hidden in a narrow view.

Reply:
It does not work, then useEffect changes to false.

Reply:
Got it.